build.gradle

```diff
@@ -275,7 +275,8 @@ dependencies {
   integrationTestCompile sourceSets.test.runtimeClasspath
   integrationTestCompile group: 'org.apache.sshd', name: 'sshd-scp', version: '2.3.0'
   integrationTestCompile group: 'org.apache.sshd', name: 'sshd-sftp', version: '2.3.0'
-  integrationTestCompile group: 'org.testcontainers', name: 'postgresql', version: '1.12.0'
+  integrationTestCompile group: 'org.testcontainers', name: 'postgresql', version: '1.12.0' // remove `commons-compress` once fixed in testcontainers
```

Comment:
weird place for that comment

Reply:
I thought so initially but then figured it's convenient when dependabot decides to bump the version and you check the diff - be extra alerted